hilti/toolchain/src/compiler/codegen/operators.cc

```diff
@@ -371,7 +404,7 @@ struct Visitor : hilti::visitor::PreOrder<cxx::Expression, Visitor> {
     /// Real
 
     result_t operator()(const operator_::real::CastToInterval& n) {
-        return fmt("::hilti::rt::Interval(%f, hilti::rt::Interval::NanosecondTag())", op0(n));
+        return fmt("::hilti::rt::Interval(%f, hilti::rt::Interval::SecondTag())", op0(n));
```

Comment:
Good catch! Is this an API breaking change or something we should backport to `release/1.5` (and possibly also `release/1.6` so we can put out a quick bugfix release)? I tend towards breaking change.

Reply:
I would say backport because it's fixing a bug.